Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -729,9 +754,10 @@ u32 VideoInterfaceManager::GetTargetRefreshRateDenominator() const
   return m_target_refresh_rate_denominator;
 }
 
-u32 VideoInterfaceManager::GetTicksPerSample() const
+double VideoInterfaceManager::GetTicksPerSample() const
```

Comment:
This was changed to a `double` because its returned value is very small, and if it was truncated to an integer right after dividing it with `m_config_vi_freq_factor` it would lose all precision.

The alternative (what was done before) would be doing this division somewhere else, but this would mean having to do the division in multiple functions instead of just this one.

Reply:
I think this does change the type of some of our operations, example would be in `GetAspectRatio()` (double -> float).  Curious if that is causing a warning on any compilers.  Does this not work as a `float`?

Reply:
There are only two calls of `GetTicksPerSample()` I think I'd prefer you do the division there instead of changing the return-type to `double`.